lib/workers/pr/changelog/release-notes.ts

```diff
@@ -260,6 +260,29 @@ export async function getReleaseNotesMd(
   return null;
 }
 
+/**
+ * Determine how long to cache release notes based on when the version was released.
+ *
+ * It's not uncommon for release notes to be updated shortly after the release itself,
+ * so only cache for about an hour when the release is less than a week old. Otherwise,
+ * cache for days.
+ */
+function releaseNotesCacheMinutes(releaseDate?: string | Date): number {
+  // for an invalid or missing release date, default to "now"
+  const releaseTS = new Date(releaseDate || '').valueOf() || Date.now();
```

Comment:
What happens if the string is wrong. Does Date throw?

Reply:
`new Date('foo').valueOf()` produces `NaN` which is falsey so false back to `Date.now()`

Reply:
and the ` || ''` was added because:
```js
new Date(null)
// > 1970-01-01T00:00:00.000Z
```